src/labels/CustomLabels.labels

```diff
@@ -1988,6 +1988,22 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <value>There are {0} One-to-One Accounts who have no Contacts.</value>
     </labels>
     <labels>
+        <fullName>healthDetailsAccountDefaultRTInvalid</fullName>
+        <categories>Health Check</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>healthDetailsAccountDefaultRTInvalid</shortDescription>
+        <value>The Record Type selected for the Account Model isn&apos;t valid.</value>
```

Comment:
Should this say "The Record Type selected for the *current* Account Model ..." (adding the word "current"). Otherwise, the message - at least to me - appears to be a bit ambiguous. What do you think @smarovich? Same for the label below (`healthDetailsAccountDefaultRTIssue`)

Reply:
That works for me, Mike!